src/components/NodeTotalChart/index.tsx

```diff
@@ -29,47 +30,98 @@ const StyledChartContainer = styled(Box)({
  * @param {Props} props
  * @returns {React.FC<Props>}
  */
-const NodeTotalChart: FC<Props> = ({ data, normalize = true }) => (
-  <StyledChartContainer>
-    <ResponsiveContainer height="100%" width="100%">
-      <BarChart
-        layout="horizontal"
-        data={data}
-        stackOffset={normalize ? "expand" : "none"}
-        maxBarSize={53}
-        barCategoryGap="8px"
-        barGap="8px"
-        aria-label="Node Total background"
-        // @ts-ignore - `overflow` is not in the type definition for BarChart
-        overflow="visible"
-      >
-        <CartesianGrid stroke="#E1E1E1" strokeWidth="0.6px" vertical={false} />
-        <Tooltip content={<NodeTooltip normalized={normalize} />} cursor={false} />
-        <YAxis
-          type="number"
-          axisLine={false}
-          tickFormatter={(tick) => formatTick(tick, normalize)}
-          domain={normalize ? [0, "dataMax"] : [0, calculateMaxDomain]}
-          interval={0}
-          tickMargin={4}
-        />
-        <XAxis
-          type="category"
-          dataKey="label"
-          axisLine={false}
-          tickLine={false}
-          tick={CustomTick}
-          interval={0}
-          allowDataOverflow
-          allowDuplicatedCategory
+const NodeTotalChart: FC<Props> = ({ data, normalize = true }) => {
+  const [tooltipData, setTooltipData] = useState<{ label: string; x: number; y: number } | null>(
+    null
+  );
+
+  const computedBarWidth = useMemo<number>(
+    () => (482 - 8 * data.length) / data.length,
```

Comment:
Could we assign variables to these numbers? Will make the logic easier to read.

Reply:
Added in 435ddca